cluster/ci/config/prow/config.yaml

```diff
@@ -2380,6 +2380,45 @@ presubmits:
         args:
           - ./hack/tf-lint.sh
 
+  - name: pull-ci-origin-installer-gotest
```

Comment:
You already have `pull-ci-origin-installer-unit`, why is this necessary?

Reply:
yes, I see that now, I see go test is called with ci-operator (https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/master.json#L44)  so I've removed that.